.ci/integrationTestSelector.groovy

```diff
@@ -90,7 +90,8 @@ pipeline {
           agent { label 'linux && immutable' }
           options { skipDefaultCheckout() }
           when {
-            expression { return (params.INTEGRATION_TEST != 'RUM') }
+            expression { return (params.INTEGRATION_TEST != 'RUM' &&
+                                 params.INTEGRATION_TEST != 'PHP') }  // TODO: When Opbeans PHP app is available then enable it
```

Comment:
NOTE: As soon as there is a PHP Opbeans project then we will be able to enable the validation